cmd/root.go

```diff
@@ -49,10 +63,21 @@ var rootCmd = &cobra.Command{
 			log.SetLevel(log.DebugLevel)
 			log.SetReportCaller(true)
 		}
+		// Configure Logrus
 		err = configureLogrusHooks()
 		if err != nil {
 			return errors.Wrap(err, "Unable to configure Logrus Hooks")
 		}
+		// Configure Honeycomb Telemetry
+		honeycombEnv, err := loadHoneycombEnv()
+		if err != nil {
+			return err
+		}
+		beeline.Init(beeline.Config{
```

Comment:
Should we only init beeline when we have a honecomb secret key? Similar to how we did with sentry?